src/utils/Fields/renderPlaceholder.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
idk it's really just a helper for making a placeholder. honestly, I think we should move away from placeholders eventually, since react has good handling of default values for select fields